src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/steps/RevealStep/partials/RevealInformationList.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, useState } from 'react';
+
+import RevealInformationItem from './RevealInformationItem';
+
+import { RevealInformationListProps } from './types';
+import { TextButton } from '~v5/shared/Button';
+import { formatText } from '~utils/intl';
+
+const RevealInformationList: FC<RevealInformationListProps> = ({ items }) => {
+  const [itemsLength, setItemsLength] = useState(6);
+
+  const itemsToShow = items.slice(0, itemsLength);
+  const shouldShowButton = items.length > 6 && itemsLength < items.length;
+
+  return (
+    <div>
+      <ul className="pt-6">
+        {itemsToShow.map(({ address, hasRevealed }) => (
+          <li key={address} className="mb-3 last:mb-0">
```

Comment:
Is it possible to use other value as a key?

Reply:
There is only address and vote inside voters object, but addresses can't be the same for users so I guess it's not bad